problems/[230619] Top K Frequent Words.js

```diff
@@ -13,7 +14,6 @@ const topKFrequent = function (words, k) {
     frequency[word] =
       frequency[word] === undefined ? 1 : (frequency[word] += 1);
```

Comment:
```suggestion
    frequency[word] = frequency[word] + 1 || 1;
```
현정님께서 저한테 남겨주신 코멘트인데 예린님 코드에도 적용 가능할 것 같아요!